copotrzebne-pl/copotrzebne-api
# feat: add opening hours

Related to: https://github.com/copotrzebne-pl/copotrzebne-api/issues/67

Test plan:
1. Add new place from postman/curl. You should provide an `openingHours` array as a property, i.e.

```json
{
    "name": "Nowy testowy punkt(test sluga z nawiasami)",
    "city": "Krakow",
    "street": "Stawowa",
    "buildingNumber": "61",
    "openingHours": [
        {
            "day": 0,
            "openTime": "8:00",
            "closeTime": "16:00"
        }
    ]
}
```

2. You should get new place with `openingHours` attached in response.
3. Update place with some new data - if you provide new opening hours for existing day (in the above case - day 0), the row will be updated, but if you provide new opening hours, new entry will be created.
4. You should get updated place with `openingHours` attached in response.
